platforms/hermes-python/hermes_python/ffi/ontology/dialogue.py

```diff
@@ -175,7 +173,39 @@ def build(cls, value, value_type):
 
     @classmethod
     def from_repr(cls, repr):
-        return cls(pointer(repr.value), repr.value_type)
+        if 1 == repr.value_type:  # CUSTOM
+            c_repr_custom_value = repr.value.value.encode('utf-8')
+            c_repr_value = cast(c_char_p(c_repr_custom_value), c_void_p)
+            return cls(c_repr_value, c_int32(repr.value_type))
+
+        elif 2 == repr.value_type:  # NUMBER
+            c_repr_number = c_double(repr.value.value)
+            cls(byref(c_repr_number), c_int32(repr.value_type))
+
+        elif 4 == repr.value_type:  # INSTANTTIME
+            c_repr_instant_time_value = CInstantTimeValue.from_repr(repr.value)
+            cls(byref(c_repr_instant_time_value), c_int32(repr.value_type))
+
+        elif 5 == repr.value_type:  # TIMEINTERVAL TODO
```

Comment:
shouldn't all these todo be resolved before merging ?

Reply:
Removed most of them, the remaining one will be for another P.R :)